src/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ find_package(HDF5_SIMPLE REQUIRED)  # the built in FindHDF5 does not handle 1.10
 find_package(FlatBuffers REQUIRED)
 find_package(h5cpp REQUIRED)
 find_package(StreamingDataTypes COMPONENTS 2246f84b16423b7ac18e901bf4c43a006fe635dc)
-find_package(GraylogLogger)
+find_package(GraylogLogger REQUIRED)
```

Comment:
This is a regression, among others because it introduces way tighter **coupling** than necessary.
It would be a better design if the different logging endpoints (like graylog, kafka-gelf, …) could be chosen optionally, while the filewriter only provides the common abstractions which are used for interfacing with the logging endpoints.
`GraylogLogger` also depends on `asio` which is another dependency that could be avoided by a more **modular** approach.